src/app/styles/DeltaStarCount.module.css

```diff
@@ -0,0 +1,7 @@
+.star {
+  margin: 0px 1px;
+}
+
+.starSpan {
+  float: right;
+}
```

Comment:
Can we stick to using Component specific names and cascade styles under it for better css rule specificity?
Ignore this if these styles are being used in multiple places.